src/renderer/views/wallet/send/SendViewBTC.tsx

```diff
@@ -59,27 +58,28 @@ export const SendViewBTC: React.FC<Props> = (props): JSX.Element => {
         isLoading={RD.isPending(txRD)}
         addressValidation={addressValidation}
         reloadFeesHandler={reloadFeesHandler}
-        fees={fees}
+        feesWithRates={fees}
       />
     ),
     [pushTx, assetsWB, txRD, addressValidation, reloadFeesHandler, fees]
   )
 
   return FP.pipe(
-    sequenceTOption(oBtcAssetWB, oExplorerUrl),
+    sequenceTOption(oClient, oBtcAssetWB),
     O.fold(
       () => <></>,
-      ([btcAssetWB, explorerUrl]) => {
-        const successActionHandler = (txHash: string) => window.apiUrl.openExternal(`${explorerUrl}tx/${txHash}`)
-
+      ([client, btcAssetWB]) => {
+        const successActionHandler = FP.flow(client.getExplorerTxUrl, window.apiUrl.openExternal)
```

Comment:
i believe this is neccessary to type `successActionHandler` as  `flow` here is outside of any `pipe` and final result is almost untyped here

Reply:
A type can be still inferred:

![hover_type](https://user-images.githubusercontent.com/61792675/98673810-2df42000-2358-11eb-9d5d-585ca4a72d8a.png)

But I can add a type of course ...